UFTBuildTask/AlmLabManagementTask/localTask.ps1

```diff
@@ -7,7 +7,6 @@ $varUserName = Get-VstsInput -Name 'varUserName' -Require
 $varPass = Get-VstsInput -Name 'varPass'
 $varDomain = Get-VstsInput -Name 'varDomain' -Require
 $varProject = Get-VstsInput -Name 'varProject' -Require
-$varRunType = Get-VstsInput -Name 'varRunType'
```

Comment:
We should keep this variable, since it used in [InvokeAlmLabManagementTaskCmdlet.cs](https://github.com/MicroFocus/ADM-TFS-Extension/blob/bb5865262bc015730edbfe6c33c6bdb5ab49b4c9/PSModule/InvokeAlmLabManagementTaskCmdlet.cs).
We can set it by default to ”TEST_SET”